diff -ruN Digest-MD5-2.12.orig/MD5.pm Digest-MD5-2.12/MD5.pm
--- Digest-MD5-2.12.orig/MD5.pm	Mon Sep 18 16:26:37 2000
+++ Digest-MD5-2.12/MD5.pm	Sat Dec  9 17:08:22 2000
@@ -15,10 +15,9 @@
 eval {
     Digest::MD5->bootstrap($VERSION);
 };
-if ($@) {
-    # Try to load the pure perl version
-    require Digest::Perl::MD5;
 
+# Try to load the pure perl version
+if ($@) { require Digest::Perl::MD5;
     Digest::Perl::MD5->import(qw(md5 md5_hex md5_base64));
     push(@ISA, "Digest::Perl::MD5");  # make OO interface work
 }
